CONTRIBUTING.md

```diff
@@ -0,0 +1,241 @@
+> You are reviewing the contributing document for getkong.org. If you want to know how to contribute to Kong itself, then please go [here](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md)
+
+# Contributing to getkong.org 📜 🦍
+
+Hello, and welcome! Whether you are looking for help, trying to report a bug,
+thinking about getting involved in the project or about to submit a patch, this
+document is for you! Its intent is to be both an entry point for newcomers to
+the community (of all technical backgrounds), and a guide/reference for
+contributors and maintainers.
+
+Consult the Table of Contents below, and jump to the desired section.
+
+## Table of Contents
+
+- [Where to seek for help?](#where-to-seek-for-help)
+- [Where to report bugs?](#where-to-report-bugs)
+- [Contributing](#contributing)
+  - [Improving the documentation](#improving-the-documentation)
+  - [Submitting a patch](#submitting-a-patch)
+    - [Git branches](#git-branches)
+    - [Commit atomicity](#commit-atomicity)
+    - [Commit message format](#commit-message-format)
+    - [Static linting](#linting)
+
+
+## Where to seek for help?
+
+See the [Kong Contributing Guideline](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-seek-for-help) for an overview of the channels at your disposal.
+
+## Where to report bugs?
+
+If the bug is about the getkong.org website, please report it in this [repository's issues tracker](https://github.com/kong/getkong.org/issues/new). If the bug is related to Kong itself, please  see the [Kong Contributing Guideline](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-report-bugs).
+
+## Contributing
+
+We welcome all kings of contributions, not only to Kong, but also to its documentation! If you wish to contribute to the documentation (by fixing a typo of making any other kind of improvement), please read the below section about [submitting a
```

Comment:
Please respect the 80 chars wide rule observed in this md file (and ideally across both Kong/kong and Kong/getkong.org)

Reply:
Only this md file? Do we have anything in place to automate this? I can keep this in mind but we might want to let others know somewhere or enforce it. `editorconfig` unfortunately doesn't support line lenght (and this project ignores *.md files in editorconfig but not sure why).

Reply:
> Only this md file?

Ideally, all. It helps enormously when reviewing diffs or vim-ing your way around.

> Do we have anything in place to automate this? 

Unfortunately not, but a good first step is to install a plugin for your editor that does it for you.

There seems to be some support for a max line length in `editorconfig`? I haven't tried it:

https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#supported-by-a-limited-number-of-editors